zipline/testing/fixtures.py

```diff
@@ -1120,24 +1121,24 @@ class WithNextAndPreviousEventDataLoader(WithPipelineEventDataLoader):
     ]
 
     next_date_intervals = [
-        [[None, '2014-01-04'],
```

Comment:
Same reason as above. Doesn't work in a straightforward way with searchsorted (when the end date is a NaT, it maps to index 0, for example). I could change it back and modify the way `get_values_for_date_ranges` works if we decide that's better than just providing concrete start/end dates.